projects/plugins/boost/app/assets/src/js/features/image-size-analysis/recommendations/table/table.module.scss

```diff
@@ -0,0 +1,4 @@
+.header-no-issues {
+	max-width: 1320px;
+	margin: 0 auto !important;
```

Comment:
`!` 😬 

is there no other way? Maybe overriding specificity with some ID at least?

Reply:
Yeah, we're using h4 and this overrides the `jb-dashboard h4` element. DO you mean add an id attribute and reference it?

Reply:
I guess so 31959fa3d95900449afa1be2940afee68114b7d4.

🥳